src/DistributionUtils.jl

```diff
@@ -51,7 +51,7 @@ end
 Generates a prior ParameterDistribution from arguments stored
 in a dictionary.
 """
-function deserialize_prior(prior_dict::Dict{String, Any})
+function deserialize_prior(prior_dict::Union{Dict{String, Any}, Dict{String, Vector{T} where T}})
```

Comment:
```suggestion
function deserialize_prior(prior_dict::Dict{String, T}) where {T}
```
Does this work? I it's perhaps not as restrictive, but I think it's a bit simpler